src/routes/(console)/project-[project]/functions/+layout.ts

```diff
@@ -8,15 +8,17 @@ import type { LayoutLoad } from './$types';
 export const load: LayoutLoad = async ({ depends }) => {
     depends(Dependencies.FUNCTION_INSTALLATIONS);
 
-    const [runtimesList, installations] = await Promise.all([
+    const [runtimesList, installations, templatesList] = await Promise.all([
         sdk.forProject.functions.listRuntimes(),
-        sdk.forProject.vcs.listInstallations([Query.limit(100)])
+        sdk.forProject.vcs.listInstallations([Query.limit(100)]),
+        sdk.forProject.functions.listTemplates()
```

Comment:
Default limit is 25 I think. We might want to do 100 here, until we introduce proper integration with server-side pagination and filtering.

(There is client side pagination currently)